draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2679,8 +2679,8 @@ e.g., the output of hash\_to\_field) may be able to carry out a dictionary attac
 To mitigate such attacks, it is recommended to first execute a more costly key
 derivation function (e.g., PBKDF2 {{!RFC2898}} or scrypt {{!RFC7914}}) on the password,
 then hash the output of that function to the target elliptic curve.
-For collision resistance, the hash underlying the key derivation function
-should be chosen according to the guidelines listed in {{hashtofield-expand}}.
+For domain separation, the hash underlying the key derivation function
+should be chosen according to the guidelines listed in {{domain-separation}}.
```

Comment:
Hmm... I think the original intent of this sentence really was about collision resistance. For example, it wouldn't be great to use a hash with 160-bit output (i.e., 80-bit collision resistance) for a cryptosystem targeting 128-bit security.

On the other hand, not totally clear to me that domain separation is a huge issue here since H isn't being used as a random oracle and the password is secret from the adversary.

I think maybe it makes sense to keep "collision resistance" and fix the target of the link... thoughts?

```suggestion
For collision resistance, the hash underlying the key derivation function
should be chosen according to the guidelines listed in {{hashtofield-expand-xmd}}.
```

Reply:
It was the link that made me think we meant something else, so we can update it. That said, should we not *also* mention domain separation here, as any KDF will surely have a hash that likely ought to be domain separated, too.

Reply:
Probably not necessary to use domain separation there because the input to the KDF is secret.

Reply:
Oh, that's a good point. Nevermind then!